charger/bender.go

```diff
@@ -172,19 +198,48 @@ func (wb *BenderCC) Status() (api.ChargeStatus, error) {
 
 // Enabled implements the api.Charger interface
 func (wb *BenderCC) Enabled() (bool, error) {
-	b, err := wb.conn.ReadHoldingRegisters(bendRegHemsCurrentLimit, 1)
-	if err != nil {
-		return false, err
+	if wb.currmilli {
+		if wb.phases != 0 {
+			b, err := wb.conn.ReadHoldingRegisters(bendRegHemsPowerLimit, 1)
+			if err != nil || binary.BigEndian.Uint16(b) == 0 {
+				return false, err
+			}
+		}
+		b, err := wb.conn.ReadHoldingRegisters(bendRegHemsCurrentLimit10, 1)
+		if err != nil || binary.BigEndian.Uint16(b) == 0 {
+			return false, err
+		}
+	} else {
+		b, err := wb.conn.ReadHoldingRegisters(bendRegHemsCurrentLimit, 1)
+		if err != nil || binary.BigEndian.Uint16(b) == 0 {
+			return false, err
+		}
 	}
 
-	return binary.BigEndian.Uint16(b) != 0, nil
+	return true, nil
 }
 
 // Enable implements the api.Charger interface
 func (wb *BenderCC) Enable(enable bool) error {
 	b := make([]byte, 2)
+	p := make([]byte, 2)
+
 	if enable {
 		binary.BigEndian.PutUint16(b, wb.current)
+		binary.BigEndian.PutUint16(p, wb.power)
+	}
+
+	if wb.currmilli {
+		_, err := wb.conn.WriteMultipleRegisters(bendRegHemsCurrentLimit10, 1, b)
+		if err != nil {
+			return err
+		}
+		if wb.phases != 0 {
```

Comment:
s.o.- in eigene Funktion packen. Wir sollten nur eins der Limits schreiben, nicht beide.